trigger.go

```diff
@@ -75,13 +75,13 @@ type Trigger struct {
 	//Value ValueType `json:""`
 
 	Opdata             string `json:"opdata,omitempty"`
-	Type               string `json:"type"`
+	Type               int    `json:"type"`
```

Comment:
I have a feeling im going to need to update the terraform project to make this change as it currently expects string types here.

I think the zabbix api expects these as string values, but we can use the json field tag to force to a string and keep the correct struct type.